src/Microsoft.AspNetCore.CookiePolicy/SameSitePolicy.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.CookiePolicy
+{
+    public enum MinimumSameSiteStrictnessPolicy
```

Comment:
Ah yes, it used to be different since there were None, LaxOrStrict, and AlwaysStrict. But now it makes sense to just name it differently.